lib/honeybadger/plugins/lambda.rb

```diff
@@ -3,12 +3,56 @@
 
 module Honeybadger
   module Plugins
+    module LambdaExtension
+      unless self.respond_to?(:hb_wrap_handler)
```

Comment:
You don't have to guard against this. 

`self.respond_to?(:hb_wrap_handler)` will always return false since we have not defined it yet, and I don't see why there would ever be a method of that name already defined at this point.

Reply:
It's a guard against the extension being included twice. But that was when we were patching it globally, so I guess I can remove that now.

Reply:
Line 7 only gets evaluated when the file is loaded, not when the module is included.